### PR TITLE
dev: add Knip and address reported issues

### PR DESCRIPTION
**Pull request by digitalcora**
[Knip](https://knip.dev) is a tool that finds unused dependencies and exports in JS apps. This adds it to our CI checks and fixes or ignores all reported issues.

Because Knip resolves modules in a slightly different way than Webpack or TypeScript do natively, this required some tweaks to configuration.

### CODE REVIEW

File: assets/src/components/admin/admin_tables.tsx [hunk 1]
```diff
@@ -417,7 +418,7 @@ const ElevatorV2ScreensTable = (): JSX.Element => {
             newRow.app_params.accessible_path_image_here_coordinates.x = value;
             return newRow;
           },
-          Cell: EditableCell,
+          Cell: EditableNumberInput,
```
Comment (PaulJKim) [inline assets/src/components/admin/admin_tables.tsx:421]: Is this a related change?

Comment (digitalcora) [reply assets/src/components/admin/admin_tables.tsx:421]: It's related in the sense that Knip reported `EditableNumberInput` as an unused export, and this caused me to realize there was actually a place we could be using it.

**Review by PaulJKim — APPROVED** [review]
neat! looks good to me
